sourced/ml/transformers/basic.py

```diff
@@ -30,12 +30,12 @@ def __call__(self, head: RDD):
         if self.keymap is None:
             return head.coalesce(self.partitions, self.shuffle)
         # partitionBy the key extracted using self.keymap
-        if self.keymap is not False:
+        else:
```

Comment:
is it a bug fix?

Reply:
No, I just thought it was more readable like this